Yoshi/Yoshi/View Controllers/Debug View Controller/DebugViewController.swift

```diff
@@ -128,8 +128,25 @@ extension DebugViewController: UITableViewDataSource {
 
         let option = options[indexPath.row]
         cell.textLabel?.text = option.title
-        cell.detailTextLabel?.text = option.subtitle
-
+        
+        if let subtitle = option.subtitle {
+            cell.detailTextLabel?.text = subtitle
+        } else {
+            switch option {
+            case let dateSelectorMenu as YoshiDateSelectorMenu:
+                cell.detailTextLabel?.text = dateSelectorMenu.selectedDate.description
+            case let tableViewMenu as YoshiTableViewMenu:                
+                for displayItem in tableViewMenu.displayItems {
+                    if displayItem.selected == true {
+                        cell.detailTextLabel?.text = displayItem.name
+                        break
+                    }
+                }
```

Comment:
We should consider using a filter here instead of a for loop, as it can probably done in 2 lines:

```
let selectedDisplayItem = tableViewMenu.displayItems.filter { $0.selected == true }.first
                cell.detailTextLabel?.text = selectedDisplayItem?.name ?? ""
```